keps/sig-architecture/20190412-conformance-behaviors.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Behavior-driven Conformance Testing
+authors:
+  - "@johnbelamaric"
+owning-sig: sig-architecture
+participating-sigs:
+  - sig-testing
+reviewers:
+  - TBD
```

Comment:
Ok. Regarding additional meta-data, we can actually get pretty far without it. Things like https://github.com/kubernetes/enhancements/pull/1099 will allow us to improve this tooling.

I am hesitant to add additional meta-data as part of this KEP, just because I want to keep the scope contained and have it not touch too much. We can add in more later. That said, I am open to the discussion.